### ISSUE TITLE
progressbar output: fail silently when python lib is unavailable

### ISSUE THREAD

**Issue by bechtoldt** [saltstack/salt#25018]
I think the execution modules concept is applicable here (HAS_IMPORT, etc.).

```
salt '*' test.ping -p
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
NameError: global name 'progressbar' is not defined
Traceback (most recent call last):
  File "/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/site-packages/salt/scripts.py", line 350, in salt_main
    client.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/salt.py", line 179, in run
    self._progress_ret(progress, out)
  File "/usr/lib/python2.7/site-packages/salt/cli/salt.py", line 271, in _progress_ret
    self.progress_bar = salt.output.get_progress(self.config, out, progress)
  File "/usr/lib/python2.7/site-packages/salt/output/__init__.py", line 64, in get_progress
    mod='output')['{0}.progress_iter'.format(out)](progress)
  File "/usr/lib/python2.7/site-packages/salt/output/progress.py", line 29, in progress_iter
    widgets = [progressbar.Percentage(), ' ', progressbar.Bar(), ' ', progressbar.Timer(), ' Returns: [', progressbar.Counter(), '/{0}]'.format(progress['minion_count'])]
NameError: global name 'progressbar' is not defined
Traceback (most recent call last):
  File "/bin/salt", line 10, in <module>
    salt_main()
  File "/usr/lib/python2.7/site-packages/salt/scripts.py", line 350, in salt_main
    client.run()
  File "/usr/lib/python2.7/site-packages/salt/cli/salt.py", line 179, in run
    self._progress_ret(progress, out)
  File "/usr/lib/python2.7/site-packages/salt/cli/salt.py", line 271, in _progress_ret
    self.progress_bar = salt.output.get_progress(self.config, out, progress)
  File "/usr/lib/python2.7/site-packages/salt/output/__init__.py", line 64, in get_progress
    mod='output')['{0}.progress_iter'.format(out)](progress)
  File "/usr/lib/python2.7/site-packages/salt/output/progress.py", line 29, in progress_iter
    widgets = [progressbar.Percentage(), ' ', progressbar.Bar(), ' ', progressbar.Timer(), ' Returns: [', progressbar.Counter(), '/{0}]'.format(progress['minion_count'])]
NameError: global name 'progressbar' is not defined
```

```
salt --versions
           Salt: 2015.5.2
         Python: 2.7.5 (default, Jun 24 2015, 00:41:19)
         Jinja2: 2.7.2
       M2Crypto: 0.21.1
 msgpack-python: 0.4.6
   msgpack-pure: Not Installed
       pycrypto: 2.6.1
        libnacl: Not Installed
         PyYAML: 3.10
          ioflo: Not Installed
          PyZMQ: 14.5.0
           RAET: Not Installed
            ZMQ: 4.0.5
           Mako: Not Installed
```

**Comment by jfindlay** [comment]
@bechtoldt, thanks for the report.  This doesn't make sense because the `__virtual__` function in that file should return `False` if `progressbar` cannot be imported.

**Comment by cachedout** [comment]
This is fixed in #25714
